### PR TITLE
chore: reissue goerli

### PR DESCRIPTION
**Pull request by simboonlong**
## Summary

- outgoing ropsten fixtures

## Changes

- collapse similar fixtures, prefix naming of `coo-`, `invoice-`, `ebl-` etc
- reissue in goerli

## Issues

Went through e2e tests and fixtures. Will want to remove the following tests, lmk if disagree:

1. `ebl-transfer-to-wallet.spec`, transfered to wallet edge case scenario, related to this [pr](https://github.com/TradeTrust/tradetrust-website/pull/284)

These are already covered by synpress flow:

1. `ebl-accepted-surrender.spec.ts`, check for accept surrendered
2. `ebl-surrendered.spec.ts`, check for surrendered
3. `ebl-wallet-owner.spec.ts`, check for connect wallet button

Considering to remove:

1. `multi-dns-verified.spec.ts`, not seen any tradetrust use case for this, seems to be inherited from opencerts

### CODE REVIEW

File: src/test/multi-dns-verified.spec.ts [hunk 1]
```diff
@@ -2,7 +2,8 @@ import { location, navigateToVerify, uploadDocument, validateIframeTexts, valida
 
 fixture("Multiple DNS Verified for Certificate Rendering").page`${location}`;
 
-test("Sample document is rendered correctly when multiple dns is verfied", async () => {
+// eslint-disable-next-line jest/no-disabled-tests
+test.skip("Sample document is rendered correctly when multiple dns is verfied", async () => {
```
Comment (simboonlong) [inline src/test/multi-dns-verified.spec.ts]: considering to remove

Comment (cavacado) [reply src/test/multi-dns-verified.spec.ts]: as discussed, since this particular case has not surfaced in the 3 years of the app's life, its probably okay to discard this test.

**Review by cavacado — APPROVED** [review]
thanks for the hard work.
I've approved but try to get another pair of eyes to look through 1 more time before merging.

File: src/utils/shared.test.ts [hunk 2]
```diff
@@ -1,34 +1,41 @@
...
 
 describe("getChainId", () => {
   it("should return the correct chainId for v2 document", () => {
     const document = {
-      ...ebl,
-      data: { ...ebl.data, network: { chain: "ETH", chainId: "3" } },
+      ...invoiceV2,
+      data: { ...invoiceV2.data, network: { chain: "ETH", chainId: "3" } },
     } as unknown as WrappedOrSignedOpenAttestationDocument;
     expect(getChainId(document)).toStrictEqual(3);
```
Comment (isaackps) [inline src/utils/shared.test.ts]: i think can change chainId to 5 or something else. cos i think near future we should be removing ropsten testnet.

File: src/utils/shared.test.ts [hunk 3]
```diff
@@ -39,27 +46,32 @@ describe("getChainId", () => {
 
   it("should return the correct chainId for v3 document", () => {
     const document = {
-      ...invoice,
+      ...invoiceV3,
       network: { chain: "ETH", chainId: "3" },
     } as unknown as WrappedOrSignedOpenAttestationDocument;
     expect(getChainId(document)).toStrictEqual(3);
```
Comment (isaackps) [inline src/utils/shared.test.ts]: same for this, should remove ropsten chainId

File: src/test/multi-dns-verified.spec.ts (hunk 1 above)
Comment (isaackps) [reply src/test/multi-dns-verified.spec.ts]: i think should remove, can maybe ask the rest of the devs.

File: package.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
     "blockchain:contracts": "node --experimental-modules ./tests/e2e/setup-contracts.mjs",
     "build": "run-s check-types build:css build:app",
     "build:dev": "cross-env NODE_ENV=development npm run build",
-    "build:test": "cross-env NODE_ENV=test NET=ropsten npm run build",
+    "build:test": "cross-env NODE_ENV=test NET=goerli npm run build",
```
Comment (superical) [inline package.json:14]: Just for info, IIRC, the`NET` is not needed because goerli is already the default now. But it doesn’t  hurt to keep it there too. :)

File: src/utils/shared.test.ts (hunk 2 above)
Comment (simboonlong) [reply src/utils/shared.test.ts]: ok updated

File: src/utils/shared.test.ts (hunk 3 above)
Comment (simboonlong) [reply src/utils/shared.test.ts]: ok updated

**Review by isaackps — APPROVED** [review]
LGTM!

File: package.json (hunk 4 above)
Comment (superical) [inline package.json:14]: Just for info, IIRC, the `NET=goerli` is optional because goerli is already the default network. But no harm leaving it here.

File: src/test/demo-sample.spec.ts [hunk 5]
```diff
@@ -9,6 +9,6 @@ test("demo sample document is rendered correctly when dragged", async (t) => {
   await t
     .expect(Selector("[data-testid='banner-title']").withText("Want to try creating a verifiable document?").exists)
     .ok();
-  await validateIssuerTexts(["EXAMPLE.OPENATTESTATION.COM"]);
+  await validateIssuerTexts(["DEMO.TRADETRUST.IO"]);
```
Comment (superical) [inline src/test/demo-sample.spec.ts:12]: Oh... Is this testing the demo doc on the frontend? 'Cos I see the fixture docs here are issued under `demo-tradetrust.openattestation.com`.

File: src/utils/shared.test.ts [hunk 6]
```diff
@@ -1,71 +1,85 @@
...
+  it("should return 'undefined' when there is no network", () => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { network, ...documentWithoutNetwork } = invoiceV3;
+
+    expect(getChainId(documentWithoutNetwork as unknown as WrappedOrSignedOpenAttestationDocument)).toStrictEqual(
+      undefined
+    );
   });
 
-  it("should throw an error when the chainId is not in the network object for v3 document", () => {
```
Comment (superical) [inline src/utils/shared.test.ts:52]: Makes sense to put them inside `describe("getChainId for v3 document")`.👍

**Review by superical — APPROVED** [review]
I think it looks alright to me. I've added some comments as I was looking through it. Nothing major. Let's put this PR through!😀🚀

File: src/test/demo-sample.spec.ts (hunk 5 above)
Comment (simboonlong) [reply src/test/demo-sample.spec.ts:12]: yes u r right its the demo doc. the demo doc i nv re-issue, keeping it as it was

File: package.json (hunk 4 above)
Comment (simboonlong) [reply package.json:14]: yup, i think will keep it there. just to be abit more explicit
